UnityProject/Assets/Editor/CompileMaterialWindow.cs

```diff
@@ -30,7 +30,7 @@ private void OnGUI()
             material.Icon = (Texture2D)EditorGUILayout.ObjectField("Cover Image", material.Icon, typeof(Texture2D), false);
 
             bool disableExportButton = false;
-            if (material.transform.Find("Pixie") == null
+            if (material.transform.Find("Wall") == null
```

Comment:
This should not be changed as it is the actual name of gameObject in the Unity editor and is not related to the filename/file extension that you intend to change.

Reply:
Will revert.